tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -465,8 +465,98 @@ def test_positive_access_rhev_with_custom_profile(self):
 
         :expectedresults: The Compute Resource created and opened successfully
 
+        :Caseautomation: Automated
+        """
+        parameter_list = [
+            ['URL', self.rhev_url, 'field'],
+            ['Username', self.rhev_username, 'field'],
+            ['Password', self.rhev_password, 'field'],
+            ['Datacenter', self.rhev_datacenter, 'special select'],
+        ]
+        name = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_resource(
+                session,
+                name=name,
+                provider_type=FOREMAN_PROVIDERS['rhev'],
+                parameter_list=parameter_list
+            )
+            self.compute_resource.set_profile_values(
+                name, COMPUTE_PROFILE_LARGE,
+                cluster=self.rhev_datacenter,
+                cores=2,
+                memory=1024,
+                network_interfaces=[
+                    dict(
+                        name='nic1',
+                        network='zstream_el6'
+                    ),
+                    dict(
+                        name='nic2',
+                        network='zstream_el7'
+                    ),
+                ],
+                storage=[
+                    dict(
+                        size='10',
+                        storage_domain='NFS-BOS',
```

Comment:
@san7ket , thanks for your review, storage_domain correspond to which field in robotello.properties file  as config data for RHEV exist but this field is missing